src/App/App.js

```diff
@@ -66,6 +71,7 @@ class App extends Component {
           view={this.state.isMovieView} 
           goHome={ this.goHome }
         />
+        {this.state.movie.error && <h3 style={{ color: 'red', textAlign: 'center' }}>{this.state.movie.error}</h3>}
```

Comment:
I added the error directly in the App, though it may make sense to add it in the AllMovies view. Again, I added the style inline with the thought it could be added to the style sheet when we get more in-depth with the CSS.